### ISSUE TITLE
Finish implementing Gherkin spec

### ISSUE THREAD

**Issue by stevenknox** [stevenknox/CoreBDD#10]

**Comment by tometchy** [comment]
Is this issue the reason why running `corebdd test --specs --output ./Specs` in _demobdd_ leaves _Divide two numbers_ scenario empty? Is it going to be implemented or it's abandoned project?

**Comment by stevenknox** [comment]
Yes, the divide by two example uses ScenarioOutline to enable data driven tests (its using xUnit Theory and InlineData under the hood) , this hasn't been implemented for gherkin generation as yet. If its a blocker happy to take a look for you or accept a pull request

**Comment by tometchy** [comment]
It's not a blocker, I just explored your demo and was wondering why it's empty. Then checked issues and here I am. Thanks for answering :)
